central/cve/converter/utils/convert_utils.go

```diff
@@ -315,6 +352,72 @@ func EmbeddedVulnerabilityToImageCVE(os string, from *storage.EmbeddedVulnerabil
 	return ret
 }
 
+// EmbeddedVulnerabilityToImageCVEV2 converts *storage.EmbeddedVulnerability object to *storage.ImageCVEV2 object
+func EmbeddedVulnerabilityToImageCVEV2(os string, imageID string, componentID string, cveIndex int, from *storage.EmbeddedVulnerability) *storage.ImageCVEV2 {
+	var nvdCvss float32
+	nvdCvss = 0
+	nvdVersion := storage.CvssScoreVersion_UNKNOWN_VERSION
+	for _, score := range from.GetCvssMetrics() {
+		if score.Source == storage.Source_SOURCE_NVD {
+			if score.GetCvssv3() != nil {
+				nvdCvss = score.GetCvssv3().GetScore()
+				nvdVersion = storage.CvssScoreVersion_V3
+
+			} else if score.GetCvssv2() != nil {
+				nvdCvss = score.GetCvssv2().GetScore()
+				nvdVersion = storage.CvssScoreVersion_V2
+			}
+		}
+	}
+	ret := &storage.ImageCVEV2{
+		Id:              cve.IDV2(from.GetCve(), imageID, componentID, strconv.Itoa(cveIndex)),
```

Comment:
Since `componentID` already has `imageID`, why  do we want `imageID` again?

Reply:
valid point.  I want it as a column for querying, but I agree we don't really need it in the ID.